src/seeders/slots/InSlotSeeder.js

```diff
@@ -78,29 +75,26 @@ class InSlotSeeder extends SlotSeeder {
 
   static generate(data = {}) {
     const slotData = super.generate(data)
-    slotData.dataType = data.dataType || faker.random.arrayElement(Object.values(InSlot.DATA_TYPES))
+    slotData.dataType = data.dataType || faker.random
+      .arrayElement([
+        FloatInSlot.DATA_TYPE,
+        IntegerInSlot.DATA_TYPE,
+        BooleanInSlot.DATA_TYPE,
+        OneOfInSlot.DATA_TYPE,
+      ])
```

Comment:
The new inslot classes need seeders, which extend the now virtual `InSlotSeeder`

it was for the annoying min max etc. behavior that is similar in float and integer sub classes that I suggested that additional intermediate layer of a virtual class (which would come with a virtual seeder). I think there is a clean way around it though - in the `Integer/FloatInSlotSeeder`s one should be able to first define the `defaultValue` and anything else that this dynamic seed random min/max value needs to run. Then when `super` is called, that can be passed, so the virtual `InSlotSeeder` can use that to derive the min/max if they aren't provided!

Reply:
Now that I have read more of the PR, it seems you already have one foot in that door, seeing how the subclasses (e.g. `FloatInSlot`) now own their min/max reality instead of the `InSlot`